docs/examples.rst

```diff
@@ -205,30 +205,30 @@ controlled by a standard PD controller. It can be seen that in this case,
 the actual position of the pendulum never converges to the desired target
 angle.
 
-   .. list-table::
-      :widths: 50 50
-      :align: center
+.. list-table::
+  :widths: 50 50
+  :align: center
 
-      * -
+  * -
 
-         .. do not delete this table row. It adds the line above the table
-            to match the one added by the row below.
+     .. do not delete this table row. It adds the line above the table
```

Comment:
I think we can remove this... the new theme seems to add a line above the table (at least, for me it did)

Reply:
This is what it looks like after I removed the first empty row. Double check that yours does the same thing too.

![image](https://user-images.githubusercontent.com/1971583/62153823-6d1d6f80-b2d3-11e9-93eb-768964e221cc.png)